analysis/wwz/get_wwz_yields.py

```diff
@@ -998,4 +998,3 @@ def main():
 
 if __name__ == "__main__":
     main()
-
```

Comment:
Could you restore this removed line? Just so that this PR does not show this file being edited (otherwise we'll look back at it in the future and may confuse ourselves that this PR changed this file).

Reply:
added back